code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1006,6 +1006,23 @@
 	if(target.incapacitated(IGNORE_GRAB) || incapacitated(IGNORE_GRAB))
 		target.visible_message(span_warning("[target] can't hang onto [src]!"))
 		return
+	
+	if(HAS_TRAIT(target, TRAIT_OVERSIZED) && !HAS_TRAIT(src, TRAIT_OVERSIZED))
+		target.visible_message(span_warning("[target] is too heavy for [src] to carry!"))
+		var/dam_zone = pick(BODY_ZONE_CHEST, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+		var/obj/item/bodypart/affecting = get_bodypart(ran_zone(dam_zone))
+		var/wound_bon = 0
+		if(!affecting) //If one leg is missing, then it might break. Snap their spine instead
+			affecting = get_bodypart(BODY_ZONE_CHEST)
+		if(prob(50)) //50% chance to break bone/rupture a muscle, etc
+			wound_bon = 100
+			to_chat(src, span_danger("You are crushed under the weight of [target]!"))
+			to_chat(target, span_danger("You accidentally crush [src]!"))
+		else
+			to_chat(src, span_danger("You hurt your [affecting.name] while trying to endure the weight of [target]!"))
+		apply_damage(25, BRUTE, affecting, wound_bonus=wound_bon) //Try to lift a 2 centner creature. 
+		Knockdown(30)
+		return
```

Comment:
You should also add modular comments to this code (`// SKYRAT EDIT START` stuff), and those numerical values (except the 0) should all be defines, so they're easier to change later.
```suggestion
		var/bonus_wound = 0
		if(!affecting) //If one leg is missing, then it might break. Snap their spine instead
			affecting = get_bodypart(BODY_ZONE_CHEST)

		if(prob(50)) //50% chance to break bone/rupture a muscle, etc
			bonus_wound = 100
			to_chat(src, span_danger("You are crushed under the weight of [target]!"))
			to_chat(target, span_danger("You accidentally crush [src]!"))
		else
			to_chat(src, span_danger("You hurt your [affecting.name] while trying to endure the weight of [target]!"))

		apply_damage(25, BRUTE, affecting, wound_bonus = bonus_wound) //Try to lift a 4-meters creature.
		Knockdown(30)
		return
```

Reply:
@GoldenAlpharex

Reply:
> Okay. I get the idea of modular and defines. But what's the problem with "2 centner"? It's more about weight, not height

What's a centner then? I thought it was just a typo and you meant meter.

Reply:
> > Okay. I get the idea of modular and defines. But what's the problem with "2 centner"? It's more about weight, not height
> 
> What's a centner then? I thought it was just a typo and you meant meter.

One centner equals to 100 kg